sample/chat-demo-mpp/common/build.gradle.kts

```diff
@@ -14,7 +14,7 @@ plugins {
 
 group = "io.github.jan.supabase"
 version = "1.0-SNAPSHOT"
-val pluginVersion = "3.2.0"
+val pluginVersion = "3.2.47-local"
```

Comment:
To be replaced by version of `supabase-kt-plugins` once it support spm4kmp for Google OAuth iOS